src/EditorFeatures/CSharpTest2/Recommendations/ByteKeywordRecommenderTests.cs

```diff
@@ -607,5 +609,20 @@ class C { }
 class C { }
 ");
         }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.KeywordRecommending)]
+        public async Task Preselection()
+        {
+            await VerifyKeywordAsync(@"
+class Program
+{
+    static void Main(string[] args)
+    {
+        Helper($$)
```

Comment:
All the intrinsic type keywords are always recommended in expression contexts, this is checking that the byte keyword recommender detects the target type of `byte` and marks the keyword for preselection.